src/Components/ServiceScene/LogsJsonScene.tsx

```diff
@@ -360,104 +438,114 @@ export class LogsJsonScene extends SceneObjectBase<LogsJsonSceneState> {
-              valueWrap={''}
-              getItemString={(_, data, itemType, itemString, keyPath) => {
-                if (data && hasProp(data, DataFrameTimeName) && typeof data.Time === 'string') {
-                  return null;
-                }
-                if (keyPath[0] === VizRootName) {
-                  return (
-                    <span className={rootNodeItemString}>
-                      {itemType} {itemString}
-                    </span>
-                  );
-                }
-
-                return <span>{itemType}</span>;
-              }}
-              valueRenderer={(valueAsString, _, keyPath) => {
-                if (keyPath === DataFrameTimeName) {
-                  return null;
-                }
-
-                return <>{valueAsString?.toString()}</>;
-              }}
-              shouldExpandNodeInitially={(_, __, level) => level <= 2}
-              labelRenderer={(keyPath, nodeType) => {
-                const nodeTypeLoc = nodeType as NodeTypeLoc;
-
-                if (keyPath[0] === VizRootName) {
-                  return model.renderNestedNodeButtons(keyPath, jsonFiltersSupported);
-                }
-
-                // Value nodes
-                if (
-                  nodeTypeLoc !== 'Object' &&
-                  nodeTypeLoc !== 'Array' &&
-                  keyPath[0] !== DataFrameTimeName &&
-                  !isLogLineField(keyPath[0].toString()) &&
-                  keyPath[0] !== VizRootName &&
-                  !isNumber(keyPath[0])
-                ) {
-                  return model.renderValueLabel(
-                    keyPath,
-                    lineField,
-                    fieldsVar,
-                    jsonParserPropsMap,
-                    jsonFiltersSupported
-                  );
-                }
-
-                // Parent nodes
-                if (
-                  (nodeTypeLoc === 'Object' || nodeTypeLoc === 'Array') &&
-                  !isLogLineField(keyPath[0].toString()) &&
-                  keyPath[0] !== VizRootName &&
-                  !isNumber(keyPath[0])
-                ) {
-                  return model.renderNestedNodeFilterButtons(
-                    keyPath,
-                    fieldsVar,
-                    jsonParserPropsMap,
-                    jsonFiltersSupported
-                  );
-                }
-
-                // Show the timestamp as the label of the log line
-                if (isNumber(keyPath[0]) && keyPath[1] === VizRootName) {
-                  const time = lineField.values[keyPath[0]]?.[DataFrameTimeName];
-                  return <strong>{time}</strong>;
-                }
-
-                // Don't render time node
-                if (keyPath[0] === DataFrameTimeName) {
-                  return null;
-                }
-
-                return <strong>{keyPath[0]}:</strong>;
-              }}
+        <div className={styles.container}>
+          {logsControlsSupported && (
```

Comment:
do we also want to check the `logsPanelControls` ff?

Reply:
Ohhh, good call!

Reply:
Checking for the featue flag now.